<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDockDetails</class>
 <widget class="QDialog" name="AdminDockDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgb(159, 208, 213)
}

QPushButton {
color: rgb(0, 0, 0);
background-color: rgb(137,198,205);
border-color: blue;
border-width: 0px;
border-style: solid;
border-radius: 7;
}

QPushButton:pressed{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 241);
}

QLabel{
color: rgb(0, 0, 0);
background-color: rgb(153, 153, 153);
selection-color: white;
selection-background-color: rgb(191, 31, 127);
border: 2px;
border-radius: 10px;
padding: 2px 4px;
}

QLineEdit {
color: rgb(127, 0, 63);
background-color: rgb(255, 255, 241);
selection-color: white;
selection-background-color: rgb(191, 31, 127);
border: 2px;
border-radius: 10px;
padding: 2px 4px;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEditContainersPerHour">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumberOfDocks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ilość doków</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumberOfDocks">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>410</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Szczegoły doków</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassengersPerHour">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ile pasażerów na godzinę</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lableContainersPerHour">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ile kontenerów na godzinę</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTransferDetails">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Transferuj zmiany</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelMaxShips">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. statków w dokach</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMaxDraft">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDockCapacity">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxDraft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Maksymalna zanurzenie</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lableDockCapacity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ilość statków na dok</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowMaxShips">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: #99CCFF;
border-color: blue;
border-width: 0px;
border-style: solid;
border-radius: 7;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassengersPerHour">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCostPerHour">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Koszt na godzinę</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCostPerHour">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
